README.md

```diff
@@ -1,93 +1,56 @@
-* [Beginner issues](https://github.com/envoyproxy/envoy/issues?q=is%3Aopen+is%3Aissue+label%3Abeginner)
-* [Build/test quick start using docker](ci#building-and-running-tests-as-a-developer)
-* [Developer guide](DEVELOPER.md)
-* Consider installing the Envoy [development support toolchain](https://github.com/envoyproxy/envoy/blob/master/support/README.md), which helps automate parts of the development process, particularly those involving code review.
-* Please make sure that you let us know if you are working on an issue so we don't duplicate work!
-
-## Community Meeting
-
-The Envoy team meets twice per month on Tuesday, alternating between 9am PT and 5PM PT. The public
-Google calendar is here: https://goo.gl/PkDijT
-
-* Meeting minutes are [here](https://goo.gl/5Cergb)
-* Recorded videos are posted [here](https://www.youtube.com/channel/UCvqbFHwN-nwalWPjPUKpvTA/videos?view=0&sort=dd&shelf_id=1)
-
-## Security
-
-### Security Audit
-
-A third party security audit was performed by Cure53, you can see the full report [here](docs/SECURITY_AUDIT.pdf).
-
-### Reporting security vulnerabilities
-
-If you've found a vulnerability or a potential vulnerability in Envoy please let us know at
-[envoy-security](mailto:envoy-security@googlegroups.com). We'll send a confirmation
-email to acknowledge your report, and we'll send an additional email when we've identified the issue
-positively or negatively.
-
-For further details please see our complete [security release process](SECURITY.md).
+# WAF extension on Envoy proxy
+
+This repository is forked from [`envoyproxy/envoy-wasm`](https://github.com/envoyproxy/envoy-wasm), and the example WASM
+extension in the envoy-wasm repository is modified to work as a Web Application Firewall(WAF) that
+can detect SQL injection. The rules for detection are aligned with ModSecurity
+rules [942100](https://github.com/coreruleset/coreruleset/blob/v3.3/dev/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf#L45) and [942101](https://github.com/coreruleset/coreruleset/blob/v3.3/dev/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf#L1458), and SQL injection is detected with methods from
+[libinjection](https://github.com/client9/libinjection).
+
+## Deployment
+From the root of the repository, build static binary of envoy proxy:
+
+```bazel build -c opt //source/exe:envoy-static```
+
+Run tests for envoy to make sure the binary has been built successfully:
+
+```bazel test //test/common/common/...```
+
+The source code for the WASM extension is in `examples/wasm`. Build the WASM module with:
+
+```bazel build //examples/wasm:envoy_filter_http_wasm_example.wasm```
+
+The WASM binary being built will be at
+`bazel-bin/examples/wasm/envoy_filter_http_wasm_example.wasm`. Make sure that the `filename` path in `examples/wasm/envoy.yaml` matches the path to the WASM binary. Then run the WASM module:
+
+``` sudo bazel-bin/source/exe/envoy-static -l trace --concurrency 1 -c `` `pwd`/examples/wasm/envoy.yaml`` ```
```

Comment:
don't need the `sudo` if you follow my recommandation for `examples/wasm/envoy-config.yaml`

Reply:
Updated in this [commit](https://github.com/dengyijia/envoy-wasm/pull/1/commits/6cfcd9373cb609b95ec42ce16c6716fcb671a7a1)